### PR TITLE
Backend/metrics

### PR DESCRIPTION
**Pull request by conorato**
Return epochs, spectrograms and some field of metadata in the classification response

### CODE REVIEW

**Comment by WilliamHarvey97** [comment]
Since I did the review IRL, nothing to say about this. From what I understand, there is only the following on our way to complete the back-end:
- Be compatible with OpenBCI session data file AND Raw SD card file. Ensure it handles Ganglion and Cyton;
- Compute metrics about sleep stage;
- Tests edge cases input values to our endpoint.

File: backend/classification/request.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    @property
+    def out_of_bed_seconds(self):
+        """timespan, in seconds, from which the subject started the recording and got out of bed"""
+        return self.wakeup - self.stream_start
+
+    @property
+    def n_epochs(self):
+        return (self.wakeup - self.bedtime) / EPOCH_DURATION
+
+    def validate(self):
```
Comment (abelfodil) [inline backend/classification/request.py:40]: I would call validate in the constructor (and make this private)

**Review by abelfodil — CHANGES_REQUESTED** [review]
Commentaires mineurs

**Review by abelfodil — APPROVED** [review]
